DigitalLearningSolutions.Web.Tests/Controllers/LearningPortal/RecommendedLearningControllerTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Controllers.LearningPortal
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Helpers;
+    using DigitalLearningSolutions.Data.Models.External.Filtered;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Controllers.LearningPortalController;
+    using DigitalLearningSolutions.Web.Helpers.ExternalApis;
+    using DigitalLearningSolutions.Web.Tests.ControllerHelpers;
+    using FakeItEasy;
+    using FluentAssertions.AspNetCore.Mvc;
+    using FluentAssertions.Execution;
+    using Microsoft.Extensions.Configuration;
+    using NUnit.Framework;
+
+    public class RecommendedLearningControllerTests
+    {
+        private const int DelegateId = 2;
+        private const int SelfAssessmentId = 1;
+        private IConfiguration configuration = null!;
+        private RecommendedLearningController controller = null!;
+        private IFilteredApiHelperService filteredApiHelperService = null!;
+        private ISelfAssessmentService selfAssessmentService = null!;
+
+        [SetUp]
+        public void Setup()
+        {
+            filteredApiHelperService = A.Fake<IFilteredApiHelperService>();
+            selfAssessmentService = A.Fake<ISelfAssessmentService>();
+            configuration = A.Fake<IConfiguration>();
+
+            controller = new RecommendedLearningController(
+                    filteredApiHelperService,
+                    selfAssessmentService,
+                    configuration
+                )
+                .WithDefaultContext()
+                .WithMockUser(true, delegateId: DelegateId);
+        }
+
+        [Test]
+        public async Task
+            SelfAssessmentResults_redirect_to_expected_action_does_not_call_filtered_api_when_using_signposting()

```

Comment:
I'm debating whether there is a point to introducing tests to make sure everything works as expected when signposting is not used. Given that there weren't any before, and the old code will probably end up unused, I think it's fine to not implement any new tests for that. Thoughts?

Reply:
Yeah that was roughly my thinking in not having them there. I'd also have to do a lot more figuring out what we need setting up all the faked API calls.
I think we should probably be covered by testing the system with the feature flag off, but we could always come back and add them if we want later if that testing all passes.